spec/02-integration/05-proxy/05-ssl_spec.lua

```diff
@@ -168,6 +169,32 @@ describe("SSL", function()
       })
       assert.res_status(426, res)
     end)
+
+    it("allows with https x-forwarded-proto from trusted client", function()
+      local res = assert(client:send {
+        method = "GET",
+        path = "/status/200",
+        headers = {
+          Host = "example.com",
+          ["x-forwarded-proto"] = "https"
+        }
+      })
+      assert.res_status(200, res)
+    end)
+
+    pending("blocks with https x-forwarded-proto from untrusted client", function()
+      -- FIXME how do we gracefully restart kong here with a separate config
+      -- that does not contain a trusted_ips block?
```

Comment:
Instead of creating yet another fixture template here, how about something like:
```lua
helpers.kong_exec("reload", {
  trusted_ips = nil -- explicitly show we don't set any, and our default is "trust none"
})
```

This would have to be in its own `describe` block, as a `setup()` hook.